cupyx/scipy/ndimage/filters.py

```diff
@@ -106,12 +107,8 @@ def _correlate_or_convolve(input, weights, output, mode, cval, origin,
         return output
     input = cupy.ascontiguousarray(input)
     weights = cupy.ascontiguousarray(weights, cupy.float64)
-    # weights is always casted to float64 in order to get an output compatible
-    # with SciPy, thought float32 might be sufficient when input dtype is low
-    # precision.
-    in_params, out_params, operation, name = _generate_correlete_kernel(
-        input.ndim, mode, cval, input.shape, wshape, origin)
-    return cupy.ElementwiseKernel(in_params, out_params, operation, name)(
+    return _get_correlete_kernel(
```

Comment:
There is a problem when memoizing a list here, this is the source of Jenkins errors